fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricCodecDataProvider.java

```diff
@@ -69,11 +105,27 @@ public CompletableFuture<?> run(DataWriter writer) {
 	 * Implement this method to register entries to generate.
 	 *
 	 * @param provider A consumer that accepts an {@link Identifier} and a value to register.
+	 * @deprecated Please use {@link FabricCodecDataProvider#configure(BiConsumer, RegistryWrapper.WrapperLookup)}.
 	 */
-	protected abstract void configure(BiConsumer<Identifier, T> provider);
+	@Deprecated(forRemoval = true)
+	protected void configure(BiConsumer<Identifier, T> provider) {
+	}
```

Comment:
This should throw, saying that the method needs to be overriden. Previously this was forced by being abstract.

Reply:
Added an `UnsupportedOperationException` with the message, not sure if that is the desired exception here though.